packages/manager/modules/adp/src/deploy/review.html

```diff
@@ -0,0 +1,138 @@
+<div class="mb-3">
+    <oui-message data-aria-close-button-label="Close" data-type="info">
+        <h6 data-translate="adp_deploy_review_info_header"></h6>
+        <p data-translate="adp_deploy_review_info_details"></p>
+    </oui-message>
+
+    <!-- cloud project -->
+    <section class="mb-3">
+        <h6 data-translate="adp_deploy_review_cloud_project_title"></h6>
+        <div class="border-top">
```

Comment:
`border-top` couldn't be set directly on the `<form>` element?

Reply:
The entire form has been replaced by `<dl>` element